refinery/analysis_manager/tasks.py

```diff
@@ -3,7 +3,7 @@
 
 @author: nils
 '''
-import urlparse
+import urllib
```

Comment:
It looks like this kind of import is not going to work after all in this case, so I'd do `from urllib.parse import urljoin`, etc.

Reply:
Can you be more specific? This works in a python shell.

Reply:
No, it doesn't:
```python
>>> import urllib
>>> urllib.parse.urlparse('foo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'urllib' has no attribute 'parse'
>>> from urllib.parse import urlparse
>>> urlparse('foo')
ParseResult(scheme='', netloc='', path='foo', params='', query='', fragment='')
```